src/Sql/Sql.Test/ScenarioTests/DatabaseCrudTests.cs

```diff
@@ -54,6 +54,13 @@ public void TestVcoreDatabaseCreateWithLicenseType()
             RunPowerShellTest("Test-CreateVcoreDatabaseWithLicenseType");
         }
 
```

Comment:
Need at least the following tests:
- Update DTU <-->VCore Serverless
- Update VCore Serverless <--> DTU
- Update mincapacity for a VCore Serverless

Reply:
Tests are added, and result copied.